packages/playground/apps/_common/components/quick-edgeless-menu.ts

```diff
@@ -303,6 +303,7 @@ export class QuickEdgelessMenu extends ShadowlessElement {
   }
 
   override render() {
+    console.log('=> service', this.rootService);
```

Comment:
Maybe removing this could fix the tests?

Reply:
Not really, we do have some other bugs.